### PR TITLE
Fetch keys from latest block instead of /keys

### PR DESCRIPTION
**Pull request by leossantiago**

### CODE REVIEW

File: src/utils/services/BlocksService.js [hunk 1]
```diff
@@ -104,13 +104,28 @@ export async function fetchLeaves(dispatch, blockId) {
...
-  get(
-    `${process.env.REACT_APP_BLOCKS_API}/keys`,
-    dispatch,
-    successCallback,
-    rejectedCallback,
-    { data: keys }
-  );
+  if (process.env.REACT_APP_SHOULD_FALLBACK) {
+    get(
+      `${process.env.REACT_APP_BLOCKS_API}/keys`,
```
Comment (avoloshko) [inline src/utils/services/BlocksService.js:109]: this endpoint does not exist anymore. how do you expect it to work?

File: src/utils/services/BlocksService.js [hunk 2]
```diff
@@ -104,13 +104,28 @@ export async function fetchLeaves(dispatch, blockId) {
...
 export async function fetchKeys(dispatch, successCallback, rejectedCallback) {
-  get(
-    `${process.env.REACT_APP_BLOCKS_API}/keys`,
-    dispatch,
-    successCallback,
-    rejectedCallback,
-    { data: keys }
-  );
+  if (process.env.REACT_APP_SHOULD_FALLBACK) {
+    get(
```
Comment (laka3000) [inline src/utils/services/BlocksService.js:108]: if it's mocked, it should be using mockRequest

**Review by laka3000 — APPROVED** [review]

File: src/utils/services/BlocksService.js (hunk 2 above)
Comment (leossantiago) [reply src/utils/services/BlocksService.js:103]: Small misunderstanding. get() method calls mockRequest() after warning about mocks being used.

File: src/utils/services/BlocksService.js (hunk 1 above)
Comment (leossantiago) [reply src/utils/services/BlocksService.js]: That's related to mocks, but already updated them as well. Thanks!

**Pull request merged**